source/slang/slang-ir-legalize-is-texture-access.cpp

```diff
@@ -0,0 +1,43 @@
+#include "slang-ir-legalize-image-subscript.h"
+
+#include "slang-ir.h"
+#include "slang-ir-insts.h"
+#include "slang-ir-util.h"
+#include "slang-ir-clone.h"
+#include "slang-ir-specialize-address-space.h"
+#include "slang-parameter-binding.h"
+#include "slang-ir-legalize-varying-params.h"
+
+namespace Slang
+{
+    void legalizeIsTextureAccess(IRModule* module)
+    {
+        IRBuilder builder(module);
+        for (auto globalInst : module->getModuleInst()->getChildren())
```

Comment:
Although not needed right now, it may make sense to extend this pass to also process insts inside generic functions. Just recurse into the inner functions if we see a IRGeneric.